tests/test_api_auth.py

```diff
@@ -137,10 +136,6 @@ def test_01_auth_with_split(self):
                 self.assertEqual(401, res.status_code, res)
                 result = res.json.get("result")
                 self.assertFalse(result.get("status"), result)
-            if six.PY2:
-                expected = "The user User(login=u'cornelius@unknown', " \
-                           "realm=u'realm1', resolver='') exists in NO resolver."
-            else:
                 expected = "The user User(login='cornelius@unknown', " \
```

Comment:
remove the init of the line, please

like this
~~~Python
with self.app.test_request_context('/auth',
                                               method='POST',
                                               data={"username": "cornelius@unknown",
                                                     "realm": "realm1",
                                                     "password": "test"}):
                res = self.app.full_dispatch_request()
                self.assertEqual(401, res.status_code, res)
                result = res.json.get("result")
                self.assertFalse(result.get("status"), result)
expected = "The user User(login='cornelius@unknown', " \
~~~

Reply:
fixed in a1e752d6c5abb37fab930e65b3c64b3680e3f3f1.